api/api-doc/column_family.json

```diff
@@ -437,6 +437,68 @@
             }
          ]
       },
+      {
+         "path":"/column_family/tombstone_gc/{name}",
```

Comment:
There's also a similar api in `/storage/service/{enable,disable}_auto_compaction`.
It's suitable to do that on a whole keyspace